shell/components/nav/TopLevelMenu.vue

```diff
@@ -273,6 +270,12 @@ export default {
       return `${ lineHeightInEm }em`;
     },
 
+    isSelectedCluster(c) {
+      const path = this.$route.path.split('/').slice()[1];
```

Comment:
This is a bit brittle (nav to fleet).

I did think it was achievable by checking the `$route`'s `product`, but that doesn't work on some routes ... and also there are other products in the explorer view.

Then doing something with checking the current route against the route of the link... but that fails again for other products in explorer view

This might be achievable piping the nuxt-link state through... but it's not something i'm 100% is possible.

Worst case scenario there is an unscoped scss block in ClusterIconMenu that works with `nuxt-link-active`

Edit... then i realised the other products in explorer is actually broken in 2.8.0. whatever comes out of that will affect this comment.